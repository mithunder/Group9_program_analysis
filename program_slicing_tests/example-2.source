module example_2 :
  read x;
  read y;
  do
    x > 0 ->
      x := x - 1
    [] x < 0 ->
      x := x + 1
  od;
  sum := 0;
  i := 0;
  do
    i < 10 ->
      sum := sum + i;
@programslicing="true"
      i := i + 1
  od;
  x := 0;
@programslicing="true"
  z := x / y;
  write z
end
