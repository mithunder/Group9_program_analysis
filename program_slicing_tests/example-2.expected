module example_2 :
  read y;
  i := 0;
  do
    i < 10 ->
@programslicing="true"
      i := i + 1
  od;
  x := 0;
@programslicing="true"
  z := x / y
end
